server/src/collection.rs

```diff
@@ -33,3 +52,31 @@ impl Collection {
         todo!()
     }
 }
+
+impl Collection {
+    pub fn new(label: String, alias: String, manager: Arc<Mutex<ServiceManager>>) -> Self {
```

Comment:
```suggestion
    pub fn new(label: &str, alias: &str, manager: Arc<Mutex<ServiceManager>>) -> Self {
```
always take a ref, and clone only where needed.

Reply:
got it, thanks!